index.html

```diff
@@ -0,0 +1,93 @@
+                alt="">
+        </div>
+        <div class="form-table margin-auto text-center">
+            <h1 class="title-form-text"> Create an Intuit account</h1>
+            <h5 class="description-form-text"> One account for everything Intuit, including Mint.</h5>
+            <a href="" class="anchor-form"> Learn more</a>
+            <div class="form-sign margin-auto">
+                <form action="/my-handling-form-page" method="post">
+                    <ul>
+                        <li>
+                            <label for="email" class="label-form">Email address</label>
+                            <input type="email" id="email" name="user_email" class="user-input">
+                        </li>
+                        <li>
+                            <label for="phone" class="label-form"> Phone (recommended)</label>
+                            <input type="number" id="phone" name="user_phone" class="phone-input">
+                            <p id="standard-call">Standard call, messaging or data rates may apply.</p>
+                        </li>
+                        <li>
+                            <label for="password" class="label-form">Password</label>
+                            <input type="password" id="password" name="user_password" class="password-input">
+                        </li>
+                    </ul>
+                </form>
+                <button type="submit" id="su-button"><i class="fas fa-lock"></i> Create Account</button>
+                <p class="end-text text-center">By clicking Create account, you agree to our <br>
+                    <a href="https://www.mint.com/terms">Terms</a>
+                    and have read and acknowledge our
+                    <a href="https://www.intuit.com/privacy/">Privacy Statement</a>. </p>
+            </div>
+        </div>
+        <div id="container-recaptcha">
+            <p class="recaptcha text-center"> Invisible reCAPTCHA by Google
+                <a href="https://www.mint.com/terms">Privacy Policy</a> and
+                <a href="https://policies.google.com/terms?hl=en">Terms of Use</a></p>
+        </div>
+    </section>
```

Comment:
- [OPTIONAL] Please consider the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document